llama_index/evaluation/benchmarks/hotpotqa.py

```diff
@@ -0,0 +1,202 @@
+import re
+from shutil import rmtree
+import string
+from typing import Any, List, Dict, Optional, Tuple
+from llama_index.indices.query.base import BaseQueryEngine
+from llama_index.indices.query.schema import QueryBundle
+from llama_index.query_engine.retriever_query_engine import RetrieverQueryEngine
+from llama_index.utils import get_cache_dir
+from llama_index.schema import NodeWithScore, TextNode
+from llama_index.indices.base_retriever import BaseRetriever
+import os
+import tqdm
+import requests
+import json
+from collections import Counter
+
+DEV_DISTRACTOR_URL = """http://curtis.ml.cmu.edu/datasets/\
+hotpot/hotpot_dev_distractor_v1.json"""
+
+
+class HotpotQAEvaluator:
```

Comment:
I wonder if the BierEvaluator and the HotpotQAEvaluator should be sharing some common interface/base class?